index.js

```diff
@@ -2,7 +2,7 @@
 
 // TODO: is this even going to resolve when distributed?
 // TODO: might want to introduce a TS build process and generate the index.d.ts
```

Comment:
Is this resolved?

Reply:
The first part is, I'll make a ticket for the second part.